libgalois/include/katana/PropertyGraph.h

```diff
@@ -592,6 +669,12 @@ class KATANA_EXPORT PropertyGraph {
 
   Result<void> SetTopology(const GraphTopology& topology);
 
+  Result<void> SetTopology(
+      std::unique_ptr<GraphTopology> topoToAssume) noexcept {
```

Comment:
Why both overloads?

Reply:
Ditto.

Reply:
One makes a copy , while the second takes over ownerhip. I've since added `&&` for second one and added a comment.